src/main/webapp/style.css

```diff
@@ -82,43 +83,66 @@ textarea:hover {
   opacity: 0.7;
 }
 
-button {
-  background-color: #856c8b;
-  border: none;
-  color: white;
-  cursor: pointer;
-  display: block;
-  font-size: 15px;
-  margin: 15px auto 25px;
-  padding: 10px 20px;
-  text-align: center;
-}
-
 header {
   background: #856c8b;
   border: 1px solid black;
   box-shadow: 0 4px 4px #00000040;
   box-sizing: border-box;
 }
 
+.topnav {
+  overflow: hidden;
```

Comment:
Out of curiosity, what overflow are you worried about here?

Reply:
It seems to have had no effect once I removed it.